src/Protocols.Abstractions/Features/IConnectionTransportFeature.cs

```diff
@@ -15,5 +15,7 @@ public interface IConnectionTransportFeature
         PipeScheduler InputWriterScheduler { get; }
 
         PipeScheduler OutputReaderScheduler { get; }
+
+        PipeScheduler ApplicationScheduler { get; set; }
```

Comment:
Why is this get;set;?

Reply:
Also, I think we should move this feature (being able to schedule the "application") to Transport abstractions. It really is a contract between Kestrel's transport and Kestrel itself. It doesn't belong in Protocol.Abstractions.